root/static/scripts/common/entity.js

```diff
@@ -34,7 +34,7 @@ import formatTrackLength from './utility/formatTrackLength';
     class Entity {
 
         constructor(data) {
-            _.assign(this, data);
+            Object.assign(this, data);
```

Comment:
As here, I should mainly only be using `Object.assign` where we want to assign into an existing object (though I may have missed some, it's not a super important distinction).